tests/test-sources/plugins/colorschemes/everforest.nix

```
@@ -0,0 +1,42 @@
+{
+  empty = {
+    colorschemes.everforest.enable = true;
+  };
+
+  defaults = {
+    colorschemes.everforest = {
+      enable = true;
+
+      settings = {
+        background = "hard";
+        enable_italic = 1;
+        disable_italic_comment = 1;
+        cursor = "red";
+        transparent_background = 1;
+        dim_inactive_windows = 1;
+        sign_column_background = "grey";
+        spell_foreground = "colored";
+        ui_contrast = "high";
+        show_eob = 0;
+        float_style = "dim";
+        diagnostic_text_highlight = 1;
+        diagnostic_line_highlight = 1;
+        diagnostic_virtual_text = "colored";
+        current_word = "bold";
+        inlay_hints_background = "dimmed";
+        disable_terminal_colors = 1;
+        lightline_disable_bold = 1;
+        colors_override = {
```

```suggestion
        # This option is broken, because the plugin tries to write its cache in /nix/store
        # better_performance = 1;
        colors_override = {
```

I'd rather have a comment rather than a silently missing option, but this is fine.

Just a preference not a request.

---

If removing it is enough to fix the test, it makes me think that `1` _is not_ the default value anyway, making me second guess some of the other defaults listed here.

---

Done! Thanks for the suggestion, definitely agree with your preference here.